### DISCUSSION TITLE
Extending sdcard adapter

### DISCUSSION THREAD

**Discussion by potatofrog** [thehighnibble/thehighnibble.github.io#39] [Q&A] [answered]
Hi All,

I am joining the club of builders that snapped off the sdcard adapter from the board. Rather than try to re solder, I wanted to try extending using the following external card with adapter: https://www.amazon.com/Adapter-Interface-Pinboard-20x20mm-Ultra-Small/dp/B0924VQGYJ/ref=sr_1_1?crid=ZV19J6TBXFAW&keywords=5pcs+TF+Card+Micro+SD+Card+Adapter+Board+Memory+Card+Interface+Pinboard+Module+20x20mm+2mm+Ultra-Small&qid=1679108432&sprefix=5pcs+tf+card+micro+sd+card+adapter+board+memory+card+interface+pinboard+module+20x20mm+2mm+ultra-small%2Caps%2C150&sr=8-1

I added a header to the mainboard, but curious the header only had eight connections and the card has nine. Tried using the ninth to ground, but everything I tried still gets the sdcard tests failed during the esp startup. Any suggestions would be appreciated.

**Comment by dmcnaugh** [comment, marked as answer]
- The 8 pins labelled `D2, D3, CMD, VD, CLK, VS, D0, D1` are the same as the 8 pin header on the IMSAI8080esp PCB, and in the same order
  - On the IMSAI8080esp PCB the top, square pad, nearest the resistor network `RN1` is the `D2` pad
- The 9th pin labelled `c-d` is the _Card Detect_ from the µSD socket, and unused/unconnected on the IMSAI8080esp PCB

**Reply by potatofrog** [reply]
My IMSAI lives!!! The key for me was align the D2, I was off by one. Thanks!
